actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -205,32 +196,7 @@ module FormHelper
       # are designed to work with an object as base, like
       # FormOptionHelper#collection_select and DateHelper#datetime_select.
       #
-      # === #form_for with a model object
-      #
-      # In the examples above, the object to be created or edited was
-      # represented by a symbol passed to +form_for+, and we noted that
-      # a string can also be used equivalently. It is also possible, however,
-      # to pass a model object itself to +form_for+. For example, if <tt>@post</tt>
-      # is an existing record you wish to edit, you can create the form using
-      #
-      #   <%= form_for @post do |f| %>
-      #     ...
-      #   <% end %>
-      #
-      # This behaves in almost the same way as outlined previously, with a
-      # couple of small exceptions. First, the prefix used to name the input
-      # elements within the form (hence the key that denotes them in the +params+
-      # hash) is actually derived from the object's _class_, e.g. <tt>params[:post]</tt>
-      # if the object's class is +Post+. However, this can be overwritten using
-      # the <tt>:as</tt> option, e.g. -
```

Comment:
It thought it was documented in the options list above, but it wasn't. I have added it in there, now that I dropped this paragraph.